pkg/pod/status_test.go

```diff
@@ -33,6 +34,63 @@ import (
 
 var ignoreVolatileTime = cmp.Comparer(func(_, _ apis.VolatileTime) bool { return true })
 
+func TestSetTaskRunStatusBasedOnStepStatus(t *testing.T) {
+	for _, c := range []struct {
+		desc              string
+		ContainerStatuses []corev1.ContainerStatus
+		want              *multierror.Error
```

Comment:
I'd remove the `want` here for now since it is `nil` for the test case here. In the future, if we wanted to add it in, we could add a `wantErr` field

Reply:
Sounds good!